ldapauthenticator/tests/test_ldapauthenticator.py

```diff
@@ -53,11 +53,20 @@ async def test_ldap_auth_disallowed(c):
     assert authorized is None
 
 
-async def test_ldap_auth_blank_template(c):
-    c.LDAPAuthenticator.bind_dn_template = [
+@pytest.mark.parametrize(
+    "bind_dn_template",
+    [
         "cn={username},ou=people,dc=planetexpress,dc=com",
-        "",
-    ]
+        ["cn={username},ou=people,dc=planetexpress,dc=com"],
+        [
+            "cn={username},ou=people,dc=planetexpress,dc=com",
+            "",
+        ],
+    ],
+)
+async def test_ldap_auth_bind_dn_template(c, bind_dn_template):
+    c.LDAPAuthenticator.bind_dn_template = bind_dn_template
+    c.LDAPAuthenticator.lookup_dn = False
```

Comment:
`lookup_dn` needs to be True here, because the `{username}` expansion in the `bind_dn_template` will without it expand to `fry` as a login name, but needs to be expanded to `Philip J. Fry` as that is the fry user's `cn` attribute.

```suggestion
```